src/frontend/app.py

```
@@ -0,0 +1,404 @@
+    "question": "Question",
+
+    "chat_id": "Chat ID",
+    "number_chat": "Chat Number",
+
+    "limit": "Limit",
+    "results_count": "Results Count",
+
+    "clean_mode": "Cleaning",
+
+    "download": "Download",
+    "search": "Search",
```

Ruff (0.15.13)

[error] Dictionary key literal `"search"` repeated

Remove repeated key literal `"search"`

(F601)